stable/lamp/templates/deployment.yaml

```diff
@@ -522,8 +522,8 @@ spec:
       {{- end }}
       {{ if .Values.sftp.enabled }}
       - name: sftp
-        image: atmoz/sftp
-        command: [/bin/bash, -c, "bash /entrypoint $SFTP_USER:$SFTP_PASSWORD:33:33:web"]
+        image: atmoz/sftp:alpine
```

Comment:
we should probably pull this image repo and version from a value so it can be pulled from a non public repo.